configs/next/packages/libnxz/sources

```diff
@@ -20,17 +20,17 @@
 #
 
 ATSRC_PACKAGE_NAME="NX GZIP library"
-ATSRC_PACKAGE_VER="v0.61"
+ATSRC_PACKAGE_VER=master
```

Comment:
On libnxz "master" is the stable branch, we should track "develop" for AT next.

Reply:
Fixed.